packages/shared/providers.yaml

```diff
@@ -5487,6 +5487,25 @@ webflow:
         base_url: https://api.webflow.com
     docs: https://docs.nango.dev/integrations/all/webflow
 
+whatsapp-business:
+    display_name: whatsapp
+    categories:
+        - communication
+        - marketing
+    auth_mode: API_KEY
+    proxy:
+        headers:
+            Authorization: Bearer ${apiKey}
+            Content-Type: application/json
+        base_url: https://graph.facebook.com
```

Comment:
could add a test no?
e.g: this seems safe https://developers.facebook.com/docs/graph-api/reference/whats-app-business-account/product_catalogs/

Reply:
updated with the latest commit, added the `/v21.0/me`